client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -67,10 +68,12 @@ class PurchasePlanDetails extends Component {
 		} );
 
 		return (
-			<div>
```

Comment:
No please. A dozen of little kittens cry every time we do it like this. 😿 

I agree it's sometimes easier to do it this way, but doing it with CSS gives us a lot more control over how it looks, and it allows us to target various device sizes and viewports separately. I'll keep the initial implementation.